src/App.tsx

```diff
@@ -0,0 +1,27 @@
+import ContentPasteIcon from '@mui/icons-material/ContentPaste';
+import Box from '@mui/material/Box';
+import { Container } from '@mui/system';
+import './App.css';
+import Patient from './views/Patient/Patient';
+
+function App(props: any) {
```

Comment:
avoid using any if we can, the props should have an interface defining what is expected